website/docs/utilities/dialog-primitive/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,23 @@
+## Usage
+
+The DialogPrimitive sub-components are not intended to be used independently of each other or in isolation. If you are looking to create your own dialog-based component, use the `DialogPrimitive::Wrapper` as your foundation.
+
+A common example of this is a non-modal dialog such as a SplitWindow or Panel which allows the user to interact with both the main page content and the content within the dialog.
+
+!!! Info
+
+If you discover a use case outside of this example, please [contact](/about/support) the HDS team for assistance.
+!!!
+
+![Example of a non-modal example](/assets/components/dialog-primitives/dialog-primitive-non-modal-example.png)
+
+### Use within HDS components
+
+Examples of how to use the DialogPrimitive components can be found in the HDS [Modal](/components/modal) and [Flyout](/components/flyout) components which are composed using the DialogPrimitive. They are both types of dialogs, but have different use cases depending on the context and type of information they convey.
+
+### What is a “dialog”
+
+The term “dialog” has somewhat of a loaded meaning and, depending on the context, can be either an ambiguous high-level term or have a specific semantic meaning. For example:
```

Comment:
(issue) Here you've given other people's definition of a dialog, but I wonder if we need to include how we are defining a dialog in this context. I think we should say what _we_ mean, even if we provide  references to other people's definition of the term.

Reply:
@jorytindall 👆

(looking at the history of the documentation this content is coming from, I think it's you who wrote this: https://github.com/hashicorp/design-system/blame/51fb2709729291cf6932fab5e90e947b11592062/website/docs/utilities/dialog-primitive/partials/guidelines/guidelines.md)

Reply:
Yeah, this is from a while back, if we were to include something more opinionated about how we term a dialog maybe I would consider adding this:

Within HDS we use the term "dialog" in reference to elements that facilitate a quick communication or exchange of information between the user and the application. We often refer to these types of experiences as "interstitial" or transitory, meaning they interrupt or break up content. Components that fall into this category include:

- Modal
- Flyout
- Tooltip
- RichTooltip
- anything else?

However, this may require more discussion if we want to take a more clear stance. I could see this being a separate workstream/ticket. What do you all think?

Reply:
I agree with @jorytindall. Since this is based on preexisting documentation, it's outside the scope of the current project. Let's discuss this in a team sync and create a separate ticket based on the outcome.

Reply:
+1 to heather and jory.

Reply:
I think what you've written above looks good but could be added as a separate follow on effort.

Reply:
I've created a quick [task](https://hashicorp.atlassian.net/jira/software/c/projects/HDS/boards/445?assignee=62a7ab87bfade200696420da&selectedIssue=HDS-3554) as a follow-on, we can discuss in sync what strategy makes sense.